ratinabox/Environment.py

```diff
@@ -87,13 +87,28 @@ def __init__(self, params={}):
         utils.update_class_params(self, self.params, get_all_defaults=True)
         utils.check_params(self, params.keys())
 
-        self.Agents : list[Agent] = []  # each new Agent will append itself to this list
-        self.agents_dict = {} # this is a dictionary which allows you to lookup a agent by name
+        self.Agents: list[Agent] = []  # each new Agent will append itself to this list
+        self.agents_dict = (
+            {}
+        )  # this is a dictionary which allows you to lookup a agent by name
```

Comment:
I assume you're using a code formatter. can this go back to being 1 line or is 
```
dict = (
{}
}
```

better for some reason I'm not aware?

Reply:
Oh dear - my autoformatter ran on this file. I didn't mean for it to. I'll update the PR to remove the autoformatting changes.

Reply:
to be honest that would be great for now but as I said below...we should probably adopt a common style :)))